pkg/R/sparkR.R

```diff
@@ -110,9 +110,19 @@ sparkR.init <- function(
                 mainClass = "edu.berkeley.cs.amplab.sparkr.SparkRBackend",
                 args = as.character(sparkRBackendPort),
                 javaOpts = paste("-Xmx", sparkMem, sep = ""))
-  Sys.sleep(2) # Wait for backend to come up
+
+  cat("Waiting JVM bring up ...\n")
+  while(TRUE) {
```

Comment:
Yeah it should be fine to double the wait time, retry 6 times and then exit (so wait times will be 1, 2, 4, 8, 16, 32 etc. giving the backend 1 minute to come up). We could add a new config option, but I'd like to avoid having too many config options if possible